src/systems/filecoin_mining/storage_mining/storage_miner_actor_state.go

```diff
@@ -238,6 +238,12 @@ func (st *StorageMinerActorState_I) _getUtilizationInfo(rt Runtime, sectorNo sec
 	return utilizationInfo
 }
 
+func (st *StorageMinerActorState_I) _getCurrUtilization(sectorNo sector.SectorNumber) block.StoragePower {
```

Comment:
there is a `_getUtilizationInfo` that does something similar in storage_miner_actor_state

Reply:
I know but it requires the runtime, which I am not sure how to pass in from the subsystem. That's what I mean above by "unsafe" sector utilization

Reply:
I think this is correct -- it should be a pure function and shouldn't need the runtime. (Why is this "unsafe"?)

Reply:
how should the actor behave if there is no corresponding sectorNo? `_getUtilizationInfo` uses runtime to handle this

Reply:
Ah, if failure is possible, then I think `_getCurrUtilization` should return `(block.StoragePower, error)`, and the caller must check the error. If the caller is actor code, it can then propagate the error via rt.Abort; if it's system code, then panic.

Reply:
@sternhenri ok, then maybe we should change `_getUtilizaitonInfo` to `_safeGetUtilizationInfo` and make it call `_getCurrUtilization` but `_getCurrUtilization` will probably need to change to `_getUtilizationInfo`?